src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
+import * as Dom from './utils/dom.js';
 import * as Events from './events';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'lodash.assign';
```

Comment:
Babel doesn't appear to translate it to something that will work (and it's not ES5)